src/lib/operators/aggregate_hash.cpp

```diff
@@ -569,6 +574,40 @@ std::enable_if_t<func == AggregateFunction::Avg && !std::is_arithmetic_v<Aggrega
   Fail("Invalid aggregate");
 }
 
+// STDDEV_SAMP writes the calculated standard deviation from current aggregate and the aggregate counter
+template <typename ColumnDataType, typename AggregateType, AggregateFunction func>
+std::enable_if_t<func == AggregateFunction::StdDevSamp && std::is_arithmetic_v<AggregateType>, void>
+write_aggregate_values(std::shared_ptr<ValueSegment<AggregateType>> segment,
+                       const AggregateResults<ColumnDataType, AggregateType>& results) {
+  DebugAssert(segment->is_nullable(), "Aggregate: Output segment needs to be nullable");
+
+  auto& values = segment->values();
+  auto& null_values = segment->null_values();
+
+  values.resize(results.size());
+  null_values.resize(results.size());
+
+  size_t i = 0;
```

Comment:
I know it is pedantic, but we normally don't use "i" as a variable name and try to come up with something more speaking. in this case "chunk_offset"?